### PR TITLE
Bugfix/multi dimension test

### PR DESCRIPTION
**Pull request by regniets**
PR for this potential bug: https://github.com/Flowpack/Flowpack.ElasticSearch.ContentRepositoryAdaptor/issues/336

### CODE REVIEW

File: Tests/Functional/Indexer/NodeIndexerDimensionsTest.php [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            'dimensions' => ['language' => ['fr']],
+            'targetDimensions' => ['language' => 'fr']
+        ]);
+
+        $rootNode = $defaultLanguageDimensionContext->getRootNode();
+        $this->siteNodeDefault = $rootNode->createNode('root', $this->nodeTypeManager->getNodeType('Neos.NodeTypes:Page'));
+        $this->siteNodeDefault->setProperty('title', 'root-default');
+        $this->siteNodeDe = $deLanguageDimensionContext->adoptNode($this->siteNodeDefault, true);
+        $this->siteNodeDe->setProperty('title', 'root-de');
+        $this->siteNodeFr = $deLanguageDimensionContext->adoptNode($this->siteNodeDefault, true);
```
Comment (DrillSergeant) [inline Tests/Functional/Indexer/NodeIndexerDimensionsTest.php:204]: Looks like $frLanguageDimensionContext should be used here?

Comment (regniets) [reply Tests/Functional/Indexer/NodeIndexerDimensionsTest.php:204]: You are right, that is a mistake

**Comment by daniellienert** [comment]
hey @regniets, @DrillSergeant built functional tests with your PR as a base and I fixed the indexing using it.
Please have a look, if 7.0.3 also fixes your issue.

Thanks a lot for the contribution.

**Comment by regniets** [comment]
Hi @daniellienert and @DrillSergeant - thanks a lot, gonna check into it and let you know if something is missing.

**Comment by daniellienert** [comment]
@regniets, Ill close this PR for now. Just open it again if something is missing in the tests.

**Pull request closed without merging**